App.js

```diff
@@ -2,7 +2,10 @@ import React from "react";
 import { H1, H3 } from "./Components/Text";
 import { Layout } from "./Components/Layout.native";
 import { ThemeProvider } from 'emotion-theming'
+import { Products } from "./Components/Products.native";
+import products from "./Fixtures/products"
 
+// TODO: update the theme with the custom fonts so rerendering is happening here instead of in text component.
```

Comment:
Will doing this also help with this warning in the tests? -> Cannot log after tests are done. Did you forget to wait for something async in your test?
    Attempted to log "Warning: An update to MainText inside a test was not wrapped in act(...).

Reply:
Did you try something like this? https://jestjs.io/docs/en/asynchronous#asyncawait

Not sure if that will help, but seems necessary since there is some async stuff happening.

Reply:
In the text file, I am trying to mock the useFont hook, testing first before it's true and then after, bc that is the update that is happening... and I tried wrapping them in the act(()=>{} to no avail. And then I also get that error in the App.js file (and I've seen it in the other two... at times) sometimes it throws sometimes it doesn't. I'm getting a sense that I need to handle that (being the font update) before all tests to avoid that warning, but I'm just not having success figuring out the best way to do that.